pages/ChatGpt.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react'
+import '@chatscope/chat-ui-kit-styles/dist/default/styles.min.css';
+import { MainContainer, ChatContainer, MessageList, Message, MessageInput, TypingIndicator } from '@chatscope/chat-ui-kit-react';
+
+// Upload your chatgpt api key here
+const API_KEY = process.env.NEXT_PUBLIC_API_KEY;
+
+const systemMessage = {
+  "role": "system", "content": "Explain things like you're talking to a software professional with 2 years of experience."
+}
```

Comment:
Good for demo 👍 Can you try to use `react-intl` to Internationalize this project? See: https://blog.logrocket.com/react-intl-internationalize-your-react-apps/  
You can create another branch to address this. @justin-kleid  Thanks!